### PR TITLE
Start adapt

### PR DESCRIPTION
**Pull request by Pozharitskiy**

### CODE REVIEW

**Comment by angelinabylitskaya** [comment]
Wrong target branch

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,4 @@
 node_modules
 assets\style.css
+assets/style.css
+assets/style.css
```
Comment (angelinabylitskaya) [inline .gitignore]: ???

File: scss/_inheritance.scss [hunk 2]
```diff
@@ -10,18 +10,70 @@
...
-  font-family: "Raleway";
-  font-weight: 100;
   font-size: 60px;
   position: absolute;
+  width: 26px;
+  height: 26px;
+  border-left: 2px solid $data-color;
+  border-bottom: 2px solid $data-color;
+  transform: rotate(45deg);
+  left: 3%;
```
Comment (angelinabylitskaya) [inline scss/_inheritance.scss]: Bad value. do not use % here

File: scss/components/_adaptive-classes.scss [hunk 3]
```diff
@@ -0,0 +1,33 @@
+@import "../variables";
```
Comment (angelinabylitskaya) [inline scss/components/_adaptive-classes.scss]: No need to import variables here

File: scss/components/_adaptive-classes.scss [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+      &-#{$breakpont} {
+        display: none;
+      }
+    }
+  }
+}
+
+.visible {
+
+  @each $breakpont,
```
Comment (angelinabylitskaya) [inline scss/components/_adaptive-classes.scss]: wrong alignment

File: scss/components/_commercial.scss [hunk 5]
```diff
@@ -41,17 +40,17 @@
 }
 
 .list-letters-three {
-  margin: 32px 0 0 114px;
+  margin: 32px 0 0 46%;
```
Comment (angelinabylitskaya) [inline scss/components/_commercial.scss]: 46% - bad value. Use % only for 100%, 50% etc

File: scss/components/_container.scss [hunk 6]
```diff
@@ -0,0 +1,16 @@
+@import "./mixins/flex-mix.scss";
```
Comment (angelinabylitskaya) [inline scss/components/_container.scss]: No need to import these files here

**Comment by angelinabylitskaya** [comment]
image

**Comment by angelinabylitskaya** [comment]
Add meta tag for responsiveness

**Comment by angelinabylitskaya** [comment]
image

File: scss/components/_adaptive-classes.scss (hunk 3 above)
Comment (Pozharitskiy) [reply scss/components/_adaptive-classes.scss]: Fixed

File: scss/components/_adaptive-classes.scss (hunk 4 above)
Comment (Pozharitskiy) [reply scss/components/_adaptive-classes.scss]: It is a work of beautify

File: scss/components/_commercial.scss (hunk 5 above)
Comment (Pozharitskiy) [reply scss/components/_commercial.scss]: Fixed

File: scss/components/_container.scss (hunk 6 above)
Comment (Pozharitskiy) [reply scss/components/_container.scss]: Done

File: scss/_inheritance.scss (hunk 2 above)
Comment (Pozharitskiy) [reply scss/_inheritance.scss]: Ok. I will fix that when i will adapt this part

**Pull request merged**